sympy/integrals/transforms.py

```diff
@@ -1825,20 +1752,36 @@ def _collapse_extra(self, extra):
                 'Laplace', None, 'No combined convergence.')
         return plane, cond
 
-    def _try_directly(self, **hints):
-        fn = self.function
-        debug('----> _try_directly: %s'%(fn, ))
+    def doit(self, **hints):
+        """
+        Try to evaluate the transform in closed form.
+
+        Explanation
+        ===========
+
+        Standard hints are the following:
+        - ``noconds``:  if True, do not return convergence conditions. The
+        default setting is `False`.
+        - ``simplify``: if True, it simplifies the final result. This is the
+        default behaviour
+        """
+        _noconds = hints.get('noconds', False)
+        _simplify = hints.get('simplify', True)
```

Comment:
Did we not decide that noconds should default to True here:
```
>>> (2*LaplaceTransform(exp(t), t, s) - 1).doit()
...
AttributeError: 'Tuple' object has no attribute 'as_coeff_Mul'
```

Reply:
Yes, we did. I threw it out while refactoring.

Reply:
Was that intentional?

Reply:
If not then it shows that a test should be added for this